scripts/release/rpm/ubi.dockerfile

```diff
@@ -17,15 +20,15 @@ COPY . .
 # RHEL 8's 'python3' is Python 3.6. RHEL 9's 'python3' is Python 3.9.
 # We have to explicitly specify 'python39' to install Python 3.9.
 RUN dos2unix ./scripts/release/rpm/azure-cli.spec && \
-    REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=$python_package PYTHON_CMD=python3.9 \
+    REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=$python_package PYTHON_CMD=$python_cmd \
     rpmbuild -v -bb --clean scripts/release/rpm/azure-cli.spec && \
     cp /root/rpmbuild/RPMS/x86_64/azure-cli-${cli_version}-1.*.x86_64.rpm /azure-cli-dev.rpm
 
 FROM ${image} AS execution-env
 
 RUN yum update -y
-RUN yum install -y python39
-
+ARG image
+RUN if [ "$image" == "centos:7" ]; then yum install -y centos-release-scl ; fi
 COPY --from=build-env /azure-cli-dev.rpm ./
-RUN rpm -i ./azure-cli-dev.rpm && \
+RUN yum install -y ./azure-cli-dev.rpm && \
```

Comment:
`rpm -i` does not install dependencies but `yum install` does, so `RUN yum install -y python39` is removed